templates/zammad/1/rancher-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: '2'
+catalog:
+  name: "Zammad"
+  version: "2.2.0-12"
+  description: "Zammad is a web based open source helpdesk/customer support system"
+  uuid: zammad-1
+  minimum_rancher_version: v1.2.0
+  questions:
+    - variable: "UPDATE_SYSCTL"
+      label: "Update host sysctl:"
+      description: |
+        Set true to avoid vm.max_map_count errors.
+        WARN: If set true, host param vm.max_map_count will be update to 262144.
+      default: false
+      required: true
+      type: enum
+      options:
+        - false
+        - true
+
+services:
+  zammad-backup:
+    scale: 1
+    start_on_create: true
+
+  zammad-elasticsearch:
+    scale: 1
+    start_on_create: true
+
+  zammad-init:
+    scale: 1
+    start_on_create: true
+
+  zammad-memcached:
+    scale: 1
+    start_on_create: true
+
+  zammad-nginx:
+    scale: 1
+    start_on_create: true
+
+  zammad-postgresql:
+    scale: 1
+    start_on_create: true
+
+  zammad-scheduler:
+    scale: 1
+    start_on_create: true
+
+  zammad-railsserver:
+    scale: 1
+    start_on_create: true
+
+  zammad-websocket:
+    scale: 1
+    start_on_create: true
+
+  zammad-nginx:
+    scale: 1
+    start_on_create: true
+    drain_timeout_ms: 10000
```

Comment:
What is this?? It's failing.
```
-    drain_timeout_ms: 10000
```

Also, what do you think to add healthcheck to zammad-nginx??
```
+    health_check:
+      healthy_threshold: 2
+      response_timeout: 2000
+      port: 80
+      unhealthy_threshold: 3
+      initializing_timeout: 300000
+      interval: 2000
+      strategy: recreate
+      request_line: GET "/" "HTTP/1.0"
```